public/static/docs/command-reference/remote/add.md

```diff
@@ -287,19 +281,13 @@ $ export OSS_ACCESS_KEY_SECRET="my-key-secret"
 $ export OSS_ENDPOINT="endpoint"
 ```
 
-#### Test your OSS storage using docker
```

Comment:
same here, it's better to have some structure

Reply:
OK, reinstated in 371400b5.